linux_os/guide/system/software/sudo/sudo_add_use_pty/rule.yml

```diff
@@ -19,6 +19,9 @@ identifiers:
     cce@rhel7: CCE-83797-1
     cce@rhel8: CCE-83798-9
 
+references:
+    cis@rhel8: 1.3.2
```

Comment:
Please add the `cis` reference together with the `anssi` entry below.

Reply:
I've addressed this above by putting the CIS reference for RHEL 8 in the right place. I noticed a CCE for RHEL 7 was already assigned, so I played the CIS reference for RHEL 7 in the references section as well.

But when it comes to adding this rule to the RHEL 7 CIS profile, I'm facing the same problem as I faced in #7058, which is that there's no obvious place to put it or way to tag it correctly since it seems like the references in that profile are outdated. `5.2.2`, which would be the current reference for this rule, is already present as a different rule:

https://github.com/ComplianceAsCode/content/blob/8ba767fedfc5e79ce72c5fdd8c31d718162b4e07/rhel7/profiles/cis.profile#L584

Reply:
The RHEL7 CIS profile will soon be realigned with latest benchmark.
So it should be fine to have the inconsistency of the rule having a cis@rhel7 reference` aligned with the latest benchmark, while the profile still mentions 2.2.0.